### ISSUE TITLE
No notification on Windows 11 when a file is being sent to you 

### ISSUE THREAD

**Issue by Pentaphon** [slowscript/warpinator-windows#14]
Any chance we can get notification support so when a file is sent to you, Windows 11 can notify you to accept it?

**Comment by slowscript** [comment]
Notifications should work, you just have to enable them in the settings. On Windows 7 it shows a balloon, on Windows 10 it shows a regular notification (although I have not yet figured out how to make it clickable). I have not tested Windows 8 and 11. If it doesn't work on 11 specifically, then it is a bug and I'll have to see what changed in 11 and how to work around it.

**Comment by Pentaphon** [comment]
> Notifications should work, you just have to enable them in the settings.

Not true. They work without having to enable them first. They are always on by default for all my apps.

**Comment by slowscript** [comment]
I meant the application's settings, although I see that is also enabled by default now that I tested it.
![settings](https://user-images.githubusercontent.com/40033124/193677572-1c059709-81b6-4095-b01b-23f5af391687.png)
I took the time to test Windows 7, 10 and even installed 11 in a virtual machine to verify it working (at least in default configuration).
![win7](https://user-images.githubusercontent.com/40033124/193678228-451e984a-3634-4e21-b3df-9587546993dd.png)
![win10](https://user-images.githubusercontent.com/40033124/193678248-c1318721-a161-4134-a5c2-455a68fa31c6.png)
![win11](https://user-images.githubusercontent.com/40033124/193678237-58ef0080-e492-488a-a224-75db9e90493d.png)
If the setting is on and it still doesn't work, maybe try deleting the configuration (under AppData/Local/slowscript). I'm busy with other projects right now, so I'll investigate this later and maybe rewrite the feature to support reacting to a click on the notification on all versions of Windows (not just 7).
